mteb/tasks/Classification/ind/IndonesianMongabayConservationClassification.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import annotations
+
+import ast
+
+import datasets
+import numpy as np
+
+from mteb.abstasks import AbsTaskClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class IndonesianMongabayConservationClassification(AbsTaskClassification):
+    metadata = TaskMetadata(
+        name="IndonesianMongabayConservationClassification",
+        description="Conservation dataset that was collected from mongabay.co.id contains topic-classification task (multi-label format) and sentiment classification. This task only covers sentiment analysis (positive, neutral negative)",
+        reference="https://aclanthology.org/2023.sealp-1.4/",
+        dataset={
+            "path": "Datasaur/mongabay-experiment",
+            "revision": "c9e9f2c09836bfec57c543ab65983f3398e9657a",
+        },
+        type="Classification",
+        category="s2s",
+        date=("2012-01-01", "2023-12-31"),
+        eval_splits=["validation", "test"],
+        eval_langs=["ind-Latn"],
+        main_score="accuracy",
```

Comment:
Might be more appropriate for imbalanced datasets.
```suggestion
        main_score="f1",
```